tests/unit/utils/test_txn_parser.py

```diff
@@ -0,0 +1,455 @@
+import copy
+from unittest import TestCase
+
+import xrpl.utils
+
+TEST_TXN = {
+    "engine_result": "tesSUCCESS",
+    "engine_result_code": 0,
+    "engine_result_message": """The transaction was applied. Only final in a validated ledger.
+""",
```

Comment:
```suggestion
    "engine_result_message": "The transaction was applied. Only final in a validated ledger.",
```
ditto for the others